package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shoutem/ui",
-  "version": "4.1.0",
+  "version": "4.1.1",
```

Comment:
Version changes are made in release branches, so if there's multiple feature branches active, they don't have to worry about which version they have to set.